We do not have this optional test dependency packaged yet.

--- pypi-json-0.4.0/tox.ini.orig
+++ pypi-json-0.4.0/tox.ini
@@ -200,7 +200,6 @@
 unused-arguments-ignore-variadic-names = True
 
 [coverage:run]
-plugins = coverage_pyver_pragma
 
 [coverage:report]
 fail_under = 99
--- pypi-json-0.4.0/tests/requirements.txt.orig
+++ pypi-json-0.4.0/tests/requirements.txt
@@ -1,7 +1,6 @@
 betamax>=0.8.1
 coincidence>=0.2.0
 coverage>=5.1
-coverage-pyver-pragma>=0.2.1
 importlib-metadata>=3.6.0
 pytest>=6.0.0
 pytest-cov>=2.8.1
